src/components/ProtectedContent/ProtectedContent.tsx

```diff
@@ -0,0 +1,61 @@
+import { AUTH_ROLE, keycloakService } from '../../services/keycloakService';
+import { ErrorPage } from '../ErrorPage/ErrorPage';
+import styles from './ProtectedContent.module.css';
+
+export enum PROTECTED_TYPE {
+  Route = 'route',
+  Component = 'component',
+}
+
+interface IProtectedContentProps {
+  type: PROTECTED_TYPE;
+  role?: AUTH_ROLE;
+  title?: string;
+  hideOnError?: boolean;
```

Comment:
even if we use naming like ErrorPage in this context, what would you say to use just `hide` in all related places?